app/helpers.php

```diff
@@ -60,3 +61,21 @@ function authenticatedOnSSOGuard(): bool
 {
     return Auth::guard(LoginController::SSO_GUARD)->check();
 }
+
+/**
+ * @param $exception Throwable?
+ * @param null $message
+ */
+function reportException($exception, $message = null): void
+{
+    if ($exception) {
```

Comment:
I'm not sure what the if statement on this achieving? if it is a null check, it will just be sending a null value to `Log:error` no?

Reply:
yep, I'm not quite sure what I was thinking either. Updated